src/components/ItemDetail/Logs.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+  <div>
+    <h3>ログ</h3>
+    <div :class="$style.itemContainer">
+      <div v-for="(log, i) in logSummaries" :key="i" :class="$style.item">

```

Comment:
```suggestion
      <div v-for="log in logSummaries" :key="log.id" :class="$style.item">
```
logSummaryにid含めるようにしてこっちに